frontend/packages/ceph-storage-plugin/src/utils/bucket-class.ts

```diff
@@ -14,3 +14,13 @@ export const validateDuration = (ms: number): boolean => ms >= 900000 && ms <= 8
 
 export const convertToMS = ({ unit, value }) =>
   unit === TimeUnits.HOUR ? parseInt(value, 10) * 3600000 : parseInt(value, 10) * 60000;
+
+export const convertMsToUnit = (unit: string, value: number): number =>
+  unit === TimeUnits.HOUR ? value / 3600000 : value / 60000;
+
+export const setProperTimeUnit = (unit: string, value: number): string => {
+  if (unit === TimeUnits.HOUR)
+    if (value === 1) return 'Hour';
+    else return 'Hours';
+  return 'Mins';
```

Comment:
```suggestion
  return 'mins';
```

Reply:
keep them in lower case

Reply:
@vbnrh And what about Hours. Should it be hour/hours or Hour/Hours?

Reply:
All of them should be in lower case :)